scripts/generate_dump

```diff
@@ -1012,6 +1012,31 @@ collect_broadcom() {
     copy_from_masic_docker "syncd" "/var/log/bcm_diag_post" "/var/log/bcm_diag_post"
 }
 
+###############################################################################
+# Collect Barefoot specific information
+# Globals:
+#  None
+# Arguments:
+#  None
+# Returns:
+#  None
+###############################################################################
+collect_barefoot() {
+    local bf_log_folder="/tmp/bf_log_folder"
+	$( rm -rf ${bf_log_folder} )
+	$( mkdir  ${bf_log_folder} )
+    unset array
+    array=( $(docker exec -ti syncd ls -1 | grep bf_drivers.log) )
+    for y in "${array[@]}"; do
+        itstr=`echo ${y} | tr -d "\r\n"`
+        copy_from_masic_docker "syncd" "/${itstr}" "${bf_log_folder}"
+    done
+
+    for file in $(find /tmp/bf_log_folder -type f); do
+        save_file "${file}" log true true
```

Comment:
In other places in this file I see the code as follows:
```
            # don't gzip already-gzipped log files :)
            if [ -z "${file##*.gz}" ]; then
                save_file $file core false
            else
                save_file $file core true
            fi
```
Shouldn't we do the same? Please check.

Reply:
This is redundant action for my case. The origin files are not zipped before.